ci-operator/config/openshift/jenkins/openshift-jenkins-release-4.12.yaml

```diff
@@ -22,7 +22,7 @@ images:
   dockerfile_path: Dockerfile.rhel8
   from: base
   inputs:
-    ocp_builder_rhel-8-golang-1.18-openshift-4.11:
+    ocp_builder_rhel-8-golang-1.19-openshift-4.12:
       as:
       - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.11
```

Comment:
you need to change it here also

Reply:
Thanks! I've changed more references, please consider the latest changes.